packages/bridge/src/vl/histogram/NumericalHistogramFilter.ts

```diff
@@ -76,7 +76,7 @@ export class NumericalHistogramFilter extends BaseHistogramFilter<Array<number |
     const min = minN instanceof Date ? `date('${minN.toISOString()}')` : minN;
     const max = maxN instanceof Date ? `date('${maxN.toISOString()}')` : maxN;
 
-    return `(@${this.columnPropName} >= ${min} and @${this.columnPropName} < ${max})`;
+    return `(@${this.columnPropName} >= ${min} and @${this.columnPropName} <= ${max})`;
```

Comment:
:boom:

Reply:
Yep, at the end it was easier than expected, but I'm afraid this might broke other things and I'd like to take a second round look 👀